src/components/presentational/DateRangeTitle/DateRangeTitle.stories.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react"
+import Layout from "../../presentational/Layout"
+import DateRangeTitle, { DateRangeTitleProps } from "./DateRangeTitle";
+import DateRangeCoordinator from "../DateRangeCoordinator";
+
+export default {
+    title: "Presentational/DateRangeTitle",
+    component: DateRangeTitle,
+    parameters: {
+        layout: 'fullscreen',
+    }
+};
+
+export const Week = {
+    args: {
+    },
+    render: (args: DateRangeTitleProps) => <Layout colorScheme="auto">
```

Comment:
Any reason not to pull out a common render method for these that just accepts an interval type.  That seems to be the pattern we typically follow.

Reply:
Not really, fixed